.github/workflows/deploy.yml

```diff
@@ -15,18 +15,18 @@ jobs:
       - uses: wagoid/commitlint-github-action@b948419dd99f3fd78a6548d48f94e3df7f6bf3ed # v6
       - uses: actions/setup-node@39370e3970a6d050c480ffad4ff0ed4d3fdee5af # v4
         with:
-          cache: "npm"
-          node-version-file: ".nvmrc"
+          cache: 'npm'
+          node-version-file: '.nvmrc'
       - name: Info
         run: |
-            cat <<EOF
-            Node version: $(node --version)
-            NPM version: $(npm --version)
-            github.event.pull_request.head.label: ${{ github.event.pull_request.head.label }}
-            github.head_ref: ${{ github.head_ref }}
-            github.ref: ${{ github.ref }}
-            github.workflow: ${{ github.workflow }}
-            EOF
+          cat <<EOF
+          Node version: $(node --version)
+          NPM version: $(npm --version)
+          github.event.pull_request.head.label: ${{ github.event.pull_request.head.label }}
+          github.head_ref: ${{ github.head_ref }}
+          github.ref: ${{ github.ref }}
+          github.workflow: ${{ github.workflow }}
+          EOF
```

Comment:
Turn on "Hide whitespace" or add "w=1" to the URL to ignore indenting changes :)